tools/importer-rest-api-specs/components/parser/cleanup/cleanup.go

```diff
@@ -142,11 +143,13 @@ func NormalizeSegment(input string, camelCase bool) string {
 		"notificationchannels":                    "notificationChannels",
 		"openshiftclusters":                       "openShiftClusters",
 		"operationresults":                        "operationResults",
+		"p2svpnGateways":                          "p2sVpnGateways",
 		"pipelineruns":                            "pipelineRuns",
 		"policysets":                              "policySets",
 		"portalconfigs":                           "portalConfigs",
 		"premieraddons":                           "premierAddons",
 		"principalassignments":                    "principalAssignments",
+		"publicipaddresses":                       "publicIpAddresses",
```

Comment:
this one's a little bit special since it's special-cased in a couple of other places - so I have a feeling this may break things in unexpected ways - if you run `make import` how's the diff look out of interest?